packages/ui/src/components/Banner/style.less

```diff
@@ -55,6 +50,63 @@
     }
   }
 
+  &-compact {
+    height: 180px;
+
+    .condo-promo-block-content-container {
+      padding: @condo-global-spacing-32 @condo-global-spacing-40 @condo-global-spacing-32 @condo-global-spacing-40;
+
+      .condo-promo-block-text-container {
+        h4.condo-typography {
+          /* stylelint-disable-next-line value-no-vendor-prefix */
+          display: -webkit-box;
+          margin-bottom: @condo-global-spacing-12;
+          overflow: hidden;
+          white-space: initial;
+          -webkit-line-clamp: 1;
+          -webkit-box-orient: vertical;
+        }
+      }
+    }
+  }
+
+  &-sm.condo-promo-block-compact,
+  &-xs.condo-promo-block-compact,
+  &-xxs.condo-promo-block-compact {
+    height: 172px;
+
+    .condo-promo-block-content-container {
+      padding: @condo-global-spacing-24 @condo-global-spacing-24 @condo-global-spacing-32;
+    }
+  }
+
+  &-xs.condo-promo-block-compact,
+  &-xxs.condo-promo-block-compact {
+    height: 196px;
+  }
+
+  &-md.condo-promo-block-no-image > &-content-container,
+  &-sm.condo-promo-block-no-image > &-content-container {
+    padding-right: @condo-global-spacing-52;
+  }
+
+  &:not(.condo-promo-block-no-image) .condo-promo-block-text-container {
+    max-width: 420px;
+  }
+
+  &-action-link {
+    display: flex;
+    align-items: center;
+
+    & > .condo-typography {
+      margin-right: @condo-global-spacing-4;
+
+      & ~ span {
+        margin-top: 3px;
```

Comment:
why 3?

Reply:
This is necessary so that the bottom of the icon is flush with the bottom of the letters.

Reply:
added specific class for icon